tests/JobCreatorTest.php

```diff
@@ -135,58 +136,65 @@ public function provideGetInstallerVersion(): array
         $currentMinorCms4 = $this->getCurrentMinorInstallerVersion('4') . '.x-dev';
         return [
             // no-installer repo
-            ['myaccount/recipe-cms', '4', ''],
-            ['myaccount/recipe-cms', '4.10', ''],
-            ['myaccount/recipe-cms', 'burger', ''],
-            ['myaccount/recipe-cms', 'pulls/4/myfeature', ''],
-            ['myaccount/recipe-cms', 'pulls/4.10/myfeature', ''],
-            ['myaccount/recipe-cms', 'pulls/burger/myfeature', ''],
-            ['myaccount/recipe-cms', '5', ''],
-            ['myaccount/recipe-cms', '5.1', ''],
-            ['myaccount/recipe-cms', '6', ''],
-            ['myaccount/recipe-cms', '6.0', ''],
+            'recipe-cms1' => ['myaccount/recipe-cms', '4', ''],
+            'recipe-cms2' => ['myaccount/recipe-cms', '4.10', ''],
+            'recipe-cms3' => ['myaccount/recipe-cms', 'burger', ''],
+            'recipe-cms4' => ['myaccount/recipe-cms', 'pulls/4/myfeature', ''],
+            'recipe-cms5' => ['myaccount/recipe-cms', 'pulls/4.10/myfeature', ''],
+            'recipe-cms6' => ['myaccount/recipe-cms', 'pulls/burger/myfeature', ''],
+            'recipe-cms7' => ['myaccount/recipe-cms', '5', ''],
+            'recipe-cms8' => ['myaccount/recipe-cms', '5.1', ''],
+            'recipe-cms9' => ['myaccount/recipe-cms', '6', ''],
+            'recipe-cms10' => ['myaccount/recipe-cms', '6.0', ''],
```

Comment:
Adding keys to these and others below was to make it easier to see what was failing while debugging this PR - I decided to keep these in so future debugging is similarly easy